src/main/java/Bob/task/Todo.java

```diff
@@ -0,0 +1,13 @@
+package bob.task;
+
+public class Todo extends Task {
+
+    public Todo(String description) {
+        super(description);
+    }
```

Comment:
I don't think this constructor is needed, is there a reason why it is implemented here?